packages/flutter_tools/templates/ffi_plugin/src.tmpl/dart-sdk/include/dart_version.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+ * for details. All rights reserved. Use of this source code is governed by a
+ * BSD-style license that can be found in the LICENSE file.
+ */
+
+#ifndef RUNTIME_INCLUDE_DART_VERSION_H_
+#define RUNTIME_INCLUDE_DART_VERSION_H_
+
+// On breaking changes the major version is increased.
+// On backwards compatible changes the minor version is increased.
+// The versioning covers the symbols exposed in dart_api_dl.h
+#define DART_API_DL_MAJOR_VERSION 2
+#define DART_API_DL_MINOR_VERSION 0
```

Comment:
This won't cover other changes to `dart_api.h` will it?

Reply:
I'll remove `dart_api_dl.h` from this PR and address it in a follow up PR after we address https://github.com/dart-lang/sdk/issues/47649 and the other issues raised in the API review.